src/dynamicprompts/jinja_extensions.py

```diff
@@ -42,19 +41,10 @@ def permutation(
 
 @pass_environment
 def wildcard(environment: Environment, wildcard_name: str) -> list[str]:
-    wm = cast(WildcardManager, environment.globals["wildcard_manager"])
-    values = []
-
-    for value in wm.get_all_values(wildcard_name):
-        if WILDCARD_RE.fullmatch(value):
-            values.extend(wildcard(environment, value))
-        elif COMBINATIONS_RE.fullmatch(value):
-            val = COMBINATIONS_RE.findall(value)[0]
-            options = val.split("|")
-            values.append(choice(options))
-        else:
-            values.append(value)
-    return values
+    generators = cast(dict[str, PromptGenerator], environment.globals["generators"])
```

Comment:
Tests fail because this would need to use the `typing.Dict` version of `dict`.

Reply:
I've removed the cast and added a mypy ignore